src/Entity/LinkOpening.php

```diff
@@ -0,0 +1,89 @@
+<?php
```

Comment:
Wouldn't it be better to merge Link and LinkOpening ? The structure would be simplier AND more importantly it would be anonymous.

Because we kinda are tracing the receivers here, plus "nbOpenings" will contain 0 or 1 most of the time